src/Sarif/Sarif.csproj

```diff
@@ -40,25 +40,26 @@
     <None Include="CodeGenHints.json" />
     <None Include="CopyrightNotice.txt" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
VS-induced change :( Who knows what it's thinking...

Reply:
Weird!